### PR TITLE
wifi: Add nRF7001DK support to lock sample

### PR DESCRIPTION
**Pull request by krish2718**

### CODE REVIEW

**Review by jukkar — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -221,7 +221,7 @@ Keys samples
 Matter samples
 --------------
 
-|no_changes_yet_note|
+* Added support for nrf7001 DK to Matter Lock Sample.
```
Comment (ioannisg) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:224]: We don't really have a 7001 DK - better refer to it with its full name (emulation board)

Comment (ArekBalysNordic) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst]: Ok, right. Is it better now?

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -221,7 +221,7 @@ Keys samples
 Matter samples
 --------------
 
-|no_changes_yet_note|
+* Support for emulation of nRF7001 Wi-Fi companion IC on nrf7002 DK to Matter Lock Sample.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:224]:
```suggestion
* :ref:`matter_lock_sample` sample:

  * Added support for emulation of the nRF7001 Wi-Fi companion IC on nRF7002 DK.
```

File: samples/matter/lock/README.rst [hunk 3]
```diff
@@ -44,7 +44,7 @@ IPv6 network support
 The development kits for this sample offer the following IPv6 network support for Matter:
 
 * Matter over Thread is supported for ``nrf52840dk_nrf52840``, ``nrf5340dk_nrf5340_cpuapp``, and ``nrf21540dk_nrf52840``.
-* Matter over Wi-Fi is supported for ``nrf5340dk_nrf5340_cpuapp`` with the ``nrf7002ek`` shield attached or for ``nrf7002dk_nrf5340_cpuapp``.
+* Matter over Wi-Fi is supported for ``nrf5340dk_nrf5340_cpuapp`` with the ``nrf7002ek`` shield attached, for ``nrf7002dk_nrf5340_cpuapp`` or for ``nrf7002dk_nrf7001_nrf5340_cpuapp`` (2.4 GHz only).
```
Comment (wiba-nordic) [inline samples/matter/lock/README.rst:47]:
```suggestion
* Matter over Wi-Fi is supported for ``nrf5340dk_nrf5340_cpuapp`` with the ``nrf7002ek`` shield attached, for ``nrf7002dk_nrf5340_cpuapp``, or for ``nrf7002dk_nrf7001_nrf5340_cpuapp`` (2.4 GHz only).
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -221,7 +221,9 @@ Keys samples
 Matter samples
 --------------
 
-|no_changes_yet_note|
+* :ref:`matter_lock_sample` sample:
+
+   * Added support for emulation of the nRF7001 Wi-Fi companion IC on nRF7002 DK.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:226]:
```suggestion
   * Added support for emulation of the nRF7001 Wi-Fi companion IC on the nRF7002 DK.
```
I missed an article.

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst (hunk 1 above)
Comment (ioannisg) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst]: yes, to me at least

**Comment by ArekBalysNordic** [comment]
Please update the sdk-connectedhomeip SHA: 9e5b1f82b43b5de78545d111ece63ce63f3024aa

**Review by ArekBalysNordic — APPROVED** [review]

**Pull request merged**